examples/src/main/scala/org/apache/spark/examples/HBaseTest.scala

```diff
@@ -36,7 +36,7 @@ object HBaseTest {
     // Initialize hBase table if necessary
     val admin = new HBaseAdmin(conf)
     if (!admin.isTableAvailable(args(0))) {
-      val tableDesc = new HTableDescriptor(args(0))
+      val tableDesc = new HTableDescriptor(TableName.valueOf(args(0)))
```

Comment:
@potix2 no it may be just fine. I was asking you to check it. It would be good to know when the new method was added to make sure this doesn't needlessly break recent versions, but, I agree with this change as long as the constructor was available in >= 0.98.7 and preferably a few previous versions.